src/app.ts

```diff
@@ -14,6 +14,8 @@ const app: PIXI.Application<ICanvas> = new PIXI.Application({
     backgroundColor: appBg,
 });
 document.body.appendChild(app.view as unknown as Node);
+// @ts-ignore
+globalThis.__PIXI_APP__ = app; // initialize Chrome extension for debugging PixiJS app
```

Comment:
add the next comment:
// TODO: remove it before deployment, because it depends on the Chrome extension

Reply:
Resolved with commit
* 93b5b3ec9566d4494bb317124acaf57e19fa81be